### PR TITLE
Relaxed javadoc rules in order to be compatible with multiple jdks

### PR DESCRIPTION
**Pull request by d-michail**
Some html tag fixed and relaxed javadoc linting in order to be compatible with newer jdks.
See #851 for discussion.

----
- [x] I read and understood <https://github.com/jgrapht/jgrapht/wiki/Become-a-Contributor>
- [x] I read and understood <https://github.com/jgrapht/jgrapht/wiki/How-to-make-your-first-%28code%29-contribution>
- [x] I added [unit tests](https://github.com/jgrapht/jgrapht/wiki/Unit-testing)
- [x] I added [documentation](https://github.com/jgrapht/jgrapht/wiki/How-to-write-documentation)
- [x] I followed the [Coding and Style Conventions](https://github.com/jgrapht/jgrapht/wiki/Coding-and-Style-Conventions)
- [x] I **have not** modified `HISTORY.md` or `CONTRIBUTORS.md`
- [x] I ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)

### CODE REVIEW

File: jgrapht-core/src/main/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTree.java [hunk 1]
```diff
@@ -45,8 +45,7 @@
  * algorithm for subset disjoint cycles by Ahuja et al. That is, the algorithm may run in
  * exponential time. This algorithm is implemented in two different version: a local search and a
  * tabu search. In both cases we have to find the best neighbor of the current capacitated spanning
- * tree. For further information about finding such an improving neighbor
- * {@link ImprovementGraph} @see ImprovementGraph
+ * tree.
```
Comment (Toptachamann) [inline jgrapht-core/src/main/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTree.java]: Why do we delete the last sentence?

File: jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java [hunk 2]
```diff
@@ -510,9 +510,6 @@ public int getComponent(V v)
          * Return the internal path array. For each vertex $v$, $pathArray[normalizeVertex(v)] = i$
          * iff $v$ appears on path $i$ or $-1$ if $v$ doesn't belong to any path.
          *
-         * <p>
-         * Note: the indexing of paths is consistent with {@link PathDecomposition#getPaths()}.
```
Comment (Toptachamann) [inline jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java]: Why do we delete this?

File: jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java [hunk 3]
```diff
@@ -536,9 +533,6 @@ public int getComponent(V v)
          * Return the internal firstNodeInPath array. For each path $i$, $firstNodeInPath[i] =
          * normalizeVertex(v)$ iff $v$ appears as the first vertex on the path.
          *
-         * <p>
-         * Note: the indexing of paths is consistent with {@link PathDecomposition#getPaths()}.
```
Comment (Toptachamann) [inline jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java]: Same question here?

File: jgrapht-core/src/main/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTree.java (hunk 1 above)
Comment (d-michail) [reply jgrapht-core/src/main/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTree.java]: The class is private, so the link is not usable. You get a warning for an undefined reference in the javadocs.

File: jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java (hunk 2 above)
Comment (d-michail) [reply jgrapht-core/src/main/java/org/jgrapht/alg/decomposition/HeavyPathDecomposition.java]: There is no such method.

**Comment by Toptachamann** [comment]
Thanks!

**Comment by HannesWell** [comment]
In total The javadoc aggregation build is now successful with OpenJDK13 at my computer.
But the Reactor summary tells me that only the JGraphT parent was successful and all other modules are skipped. Is this intended?

**Comment by d-michail** [comment]
Yes. This goal only aggregates the javadocs and produces the `target/site/apidocs` folder.

**Pull request merged**